api/src/main/java/edu/cornell/mannlib/vitro/webapp/auth/policy/PolicyHelper.java

```diff
@@ -46,84 +41,77 @@
  * are authorized by current policy.
  */
 public class PolicyHelper {
-	private static final Log log = LogFactory.getLog(PolicyHelper.class);
+    private static final Log log = LogFactory.getLog(PolicyHelper.class);
+    private static UserAccount nullUserAccount = new UserAccount();
```

Comment:
Should we use here Null object pattern?

Reply:
I think we should at some point, but we don't have an interface or an abstract class. 
Refactoring UserAccount class at this point could result in a lot of changes and might be out of scope for this PR.
So I created static instance that serves the same purpose until we address this issue in a separate PR.

Reply:
ok, let's keep as is at the moment